ios/Client/EXAppDelegate.m

```diff
@@ -12,6 +12,10 @@
 #import "EXRootViewController.h"
 #import "EXConstants.h"
 
+#if __has_include(<GoogleSignIn/GoogleSignIn.h>)
```

Comment:
Is this module only going to be used in standalone apps? If so, shouldn't this code go rather to `exponent-view-template`? Or is it only for testing purposes?

Reply:
I don't know how to do this, where is `exponent-view-template` located?